pages/index.tsx

```diff
@@ -290,6 +293,15 @@ const Status = (props: SubgraphIndexingStatus) => {
                 </span>
               </>
             )}
+            {' 📈 '}
+
+            <a
+            target="_blank"
+            className="text-purple-600 hover:underline"
+            href={`https://subgraphs.messari.io/subgraph?endpoint=https://api.thegraph.com/subgraphs/id/${props.subgraph}&tab=protocol`}
+            >
+            Visual
```

Comment:
To be explicit let's name it "Messari Visualizer"? what do you think
Otherwise, when people click on the link for a non-Messari subgraph they would think this is a bug, not a feature 🤣